### PR TITLE
Nivelate Gradle & Maven

### PR DESCRIPTION
**Pull request by jabrena**
Small PR to avoid problems for Gradle users
Fixed the issue to run in local the Azure function

### CODE REVIEW

**Comment by andxu** [comment]
Hi @jabrena , thank you for your PR, the changes in your PR seems excellent except one version downgrade from 1.1.0 to 1.0.0-RC(actually we published 1.2.0 today), would you please resolve the conflicts and update the version ? thanks again.

**Comment by jabrena** [comment]
Hi @andxu, I have upgraded the `build.gradle` to have the same plugin version than in Master.

File: build.gradle [hunk 1]
```diff
@@ -1,32 +1,34 @@
 plugins {
-  id "com.microsoft.azure.azurefunctions" version "1.1.0"
+	id "com.microsoft.azure.azurefunctions" version "1.2.0"
```
Comment (Flanker32) [inline build.gradle:2]: use space instead of tab?

File: pom.xml [hunk 2]
```diff
@@ -11,114 +11,58 @@
     <name>Azure Java Functions</name>
 
     <properties>
+        <java.version>1.8</java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
-        <azure.functions.maven.plugin.version>1.4.1</azure.functions.maven.plugin.version>
+        <azure.functions.maven.plugin.version>1.3.4</azure.functions.maven.plugin.version>
```
Comment (Flanker32) [inline pom.xml:16]: The latest maven plugin version is 1.4.1

File: pom.xml [hunk 3]
```diff
@@ -11,114 +11,58 @@
...
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.22.2</version>
+            </plugin>
             <plugin>
                 <groupId>com.microsoft.azure</groupId>
                 <artifactId>azure-functions-maven-plugin</artifactId>
+                <version>1.4.1</version>
```
Comment (Flanker32) [inline pom.xml:65]: could use `${azure.functions.maven.plugin.version}` here

**Comment by jabrena** [comment]
Hi @Flanker32, 

I will do that changes today. 
Many thanks for the comments

File: build.gradle (hunk 1 above)
Comment (jabrena) [reply build.gradle]: The difference was: 

The original code had 2 spaces and mine 4 spaces. We should use 4 spaces in my humble opinion.

File: pom.xml (hunk 2 above)
Comment (jabrena) [reply pom.xml]: Upgraded

File: pom.xml (hunk 3 above)
Comment (jabrena) [reply pom.xml]: Upgraded

**Comment by jabrena** [comment]
Done

**Comment by jabrena** [comment]
Hi @Flanker32, @andxu I did the changes. 
Is it possible to merge?

**Comment by Flanker32** [comment]
@jabrena Thanks a lot for your contribution and really sorry for the late response, we have fixes these issues in another pr #12 with your fixes, thanks again for your help!

**Pull request closed without merging**